ros_buildfarm/templates/release/binarydeb_task.Dockerfile.em

```diff
@@ -61,6 +61,13 @@ RUN echo "@today_str"
     os_code_name=os_code_name,
 ))@
 
+@[if build_environment_variables]@
+@(TEMPLATE(
+    'snippet/set_environment_variables.Dockerfile.em',
+    environment_variables=build_environment_variables,
+))@
+@[end if]@
```

Comment:
Two avoid potentially two consecutive empty lines in the output the block should include either the leading empty line or the trailing empty line, e.g.:

```
))@
@[if build_environment_variables]@

@(TEMPLATE(
    'snippet/set_environment_variables.Dockerfile.em',
    environment_variables=build_environment_variables,
))@
@[end if]@

@(TEMPLATE(
```

Reply:
This is addressed in 41c9884. It's not automatically collapsed because the change was on a line above.